Composer/packages/server/src/directline/middleware/createUploadHandler.ts

```diff
@@ -5,12 +5,13 @@ import { StatusCodes } from 'http-status-codes';
 import { AttachmentData, Attachment } from 'botframework-schema';
 import * as express from 'express';
 import * as Formidable from 'formidable';
+import formatMessage from 'format-message';
```

Comment:
format-message is not really set up on the server I don't think. It may be though.

Reply:
Its not from what I see. I did see a few usages of it on the server. Just wrapped it in case.